sasl_defs.c

```diff
@@ -87,6 +87,7 @@ static int sasl_getconf(void *context, const char **path)
 {
     *path = getenv("SASL_CONF_PATH");
 
+#if defined(HAVE_SASL_CB_GETCONF)
     if (*path == NULL) {
```

Comment:
SASL_CONF_PATH environment variable should always override for both GETCONF and GETCONFPATH. This disables the environment variable if GETCONF is set.

Reply:
I have picked up if (*path == NULL) out of the macro check to avoid this